agent/acs/handler/attach_eni_handler_common.go

```diff
@@ -42,10 +44,26 @@ func (handler *ackTimeoutHandler) handle() {
 	}
 	if !eniAttachment.IsSent() {
 		seelog.Warnf("Timed out waiting for ENI ack; removing ENI attachment record with MAC address: %s", handler.mac)
+		handler.removeENIAttachment(handler.mac)
```

Comment:
Is it possible to change the name of this function to something like `removeAttachmentFromDataClient`, it's difficult to say what is the difference between `removeEniAttachment` and `RemoveENIAttachment` in the next line.

Reply:
Agreed, renamed to removeENIAttachmentFromData.